### PR TITLE
add apikey for symbole (.snupkg)

### PR DESCRIPTION
**Pull request by fle-bach**

### CODE REVIEW

File: build.cake [hunk 1]
```diff
@@ -84,7 +84,8 @@ Task("PushPackage")
             DotNetCoreNuGetPush(package, new DotNetCoreNuGetPushSettings 
             {
                 Source = "https://www.nuget.org/api/v2/package",
-                ApiKey = apiKey
+                ApiKey = apiKey,
+                SymbolApiKey = apiKey
```
Comment (Julien-Molina) [inline build.cake]: I think you need to specify -SymbolSource as well : https://docs.microsoft.com/fr-fr/nuget/tools/cli-ref-push

Comment (Julien-Molina) [reply build.cake]: SymbolSource="https://www.nuget.org/api/v2/symbolpackage"

Comment (Julien-Molina) [reply build.cake]: https://docs.microsoft.com/en-us/nuget/api/symbol-package-publish-resource

Comment (fle-bach) [reply build.cake]: thanks i forgot this i do it

**Pull request merged**